src/Maker/MakeSubscriber.php

```diff
@@ -115,4 +120,43 @@ public function generate(InputInterface $input, ConsoleStyle $io, Generator $gen
     public function configureDependencies(DependencyBuilder $dependencies): void
     {
     }
+
+    /**
+     * @param $event
+     */
+    private function getEventConstant($event): string
```

Comment:
```suggestion
    private function getEventConstant(string $event): string
```

And remove `@param`.